Survey/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  HomeView.swift
+//  SurveyTests
+//
+//  Created by Khanh on 07/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+struct HomeView: View {
```

Comment:
The PR and the issue you are working on are only for Unit/UI Tests, not sure why you include HomeView in this PR as well 🤔 . Should you be referencing another issue with HomeView (ideally move these code to another PR to keep at most one issue per PR.).

Reply:
That is fine. You can also create a more minimum View (with out ViewModel) to save on line changes.

Please add in PR comment on why some feature is beyond the issue next time 👍

Reply:
Thank you, I will do it next time 😄